src/Controller/AdminController.php

```diff
@@ -892,6 +893,9 @@ public function taxationSettingsAction(
     public function tagsAction(Request $request, TagManager $tagManager)
     {
         if ($request->isMethod('POST') && $request->request->has('delete')) {
+            if (!$this->isGranted('ROLE_ADMIN')) {
+                throw new AccessDeniedException();
+            }
```

Comment:
Can be simplified to `$this->denyAccessUnlessGranted('ROLE_ADMIN');` (see [docs](https://symfony.com/doc/5.4/security/voters.html))